demos/common/export_models/README.md

```diff
@@ -1,194 +1 @@
-# Exporting GEN AI Models {#ovms_demos_common_export}
```

Comment:
moving this script to different location will cause conflicts for severl open PRs, can this be considered for the future?